internal/models/models.go

```diff
@@ -0,0 +1,22 @@
+package models
+
+import "gorm.io/gorm"
+
+type Bundle struct {
+	gorm.Model
+	Name string
+}
+
+type Log struct {
+	gorm.Model
+}
+
+type Publisher struct {
+	gorm.Model
+	Name string `json:"name"`
```

Comment:
Volendo gorm neanche vuole il nome (mi riferisco alla specifica json:name) se rispecchia esattamente il nome del campo (lo traduce automaticamente), addirittura fa lo snake case automatico.

Reply:
Uhm, this was an experiment because I expected gorm to do it, but it didn't. It marshalls the field as `Name`, and even `gorm.Model` that adds common fields like `UpdatedAt` and `DeletedAt`, results in upcased keys in json :thinking: 

Maybe I was missing something, because if we want `updated_at` or `deleted_at`, we need to write all the boilerplate for it, without using `gorm.Model`, but that would look awkward

Reply:
That's really strange! I will try it!